### DISCUSSION TITLE
JS data URI -> Blob

### DISCUSSION THREAD

**Discussion by andreiciobanica** [Forceu/Gokapi#157] [Q&A] [answered]
Hello, I'm seeking for some help because really I don't know what causes the: `Bad Request and the error message: "no multipart boundary param in Content-Type".

Here's my code:
https://hastebin.de/cuneqohihu.js

**Comment by andreiciobanica** [comment, marked as answer]
Seems that specifying `'Content-Type': 'multipart/form-data'` concludes to that error.

Sadly :(

**Comment by Forceu** [comment]
What exactly are you trying to do? If you want to upload something with a script, using the API would be a better solution

**Reply by andreiciobanica** [reply]
That's what I was trying to do. Now with that code it works, but without specifying in the header the `content-type`.
